.ci/updatecli.d/update-gherkin-specs.yml

```diff
@@ -62,10 +70,13 @@ actions:
       description: |-
         ### What
         APM agent Gherkin specs automatic sync
+
         ### Why
         *Changeset*
+        * {{ source "pull_request" }}
```

Comment:
[question] does this means "get the output of the `pull_request` item above" ? If yes, then that means just injecting a URL link to the PR.